pgql-lang/src/test/java/oracle/pgql/lang/MetadataTest.java

```diff
@@ -1008,102 +1005,17 @@ public void testOneRowPerEdge() throws Exception {
-    allProperties = getVertexProperties(result);
-    expectedProperties = new ArrayList<>();
-    expectedProperties.add("firstName");
-    expectedProperties.add("dob");
-    expectedProperties.add("numericProp");
-    expectedProperties.add("typeConflictProp");
-    expectedProperties.add("name");
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH (n) WHERE has_label(n, 'Person') OR has_label(n, 'University')");
-    assertEquals(expectedProperties, allProperties);
-  }
-
-  @Test
-  public void testGetAllEdgePropertiesApi() throws Exception {
-    PgqlResult result = parse("SELECT 1 FROM MATCH () -[e:worksFor]-> () ON financialNetwork");
-    List<String> allProperties = getEdgeProperties(result);
-    assertEquals(Collections.EMPTY_LIST, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e:worksFor|transaction]-> () ON financialNetwork");
-    allProperties = getEdgeProperties(result);
-    List<String> expectedProperties = new ArrayList<>();
-    expectedProperties.add("amount");
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> () ON financialNetwork WHERE has_label(e, 'worksFor')");
-    allProperties = getEdgeProperties(result);
-    expectedProperties = new ArrayList<>();
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> () ON financialNetwork WHERE has_label(e, 'transaction')");
-    allProperties = getEdgeProperties(result);
-    expectedProperties = new ArrayList<>();
-    expectedProperties.add("amount");
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> () ON financialNetwork WHERE has_label(e, 'TRANSACTION')");
-    allProperties = getEdgeProperties(result);
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> () ON financialNetwork WHERE \"has_label\"(e, 'transaction')");
-    allProperties = getEdgeProperties(result);
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> () ON financialNetwork");
-    allProperties = getEdgeProperties(result);
-    assertEquals(expectedProperties, allProperties);
-
-    result = parse("SELECT 1 FROM MATCH () -[e]-> ()");
-    allProperties = getEdgeProperties(result);
-    expectedProperties = new ArrayList<>();
-    expectedProperties.add("since");
-    expectedProperties.add("prop");
-    expectedProperties.add("typeConflictProp");
-    expectedProperties.add("PROP");
-    expectedProperties.add("Typeconflictprop");
-    assertEquals(expectedProperties, allProperties);
-  }
-
-  private List<String> getEdgeProperties(PgqlResult result) {
-    QueryVariable edge = result.getGraphQuery().getGraphPattern().getConnections().iterator().next();
-    return result.getAllProperties(edge);
-  }
+  public void testOneRowPerStep() throws Exception {
```

Comment:
we had a test like this for ONE ROW PER VERTEX and ONE ROW PER EDGE, but one for ONE ROW PER STEP was missing